template/typeahead/typeahead-match.html

```diff
@@ -1 +1,4 @@
-<a href tabindex="-1" ng-bind-html="match.label | uibTypeaheadHighlight:query"></a>
+<a href
+   tabindex="-1"
+   ng-bind-html="match.label | uibTypeaheadHighlight:query"
+   ng-attr-title="{{match.label}}"></a>
```

Comment:
[These guys](http://www.thinkingmedia.ca/2015/03/conditionally-add-a-html-element-attribute-value-in-angularjs/#understanding-the-ng-attr-prefix) explain it pretty well.

It is basically so you never see the text ```{{match.label}}``` in the tooltip. This will only add the title attribute with the actual value of match.label.